test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -743,7 +743,7 @@ public void testEnforceSnapshotSync() throws Exception {
         txKey = LogReplicationMetadataKey.newBuilder().setKey(LAST_SNAPSHOT_STARTED.getVal()).build();
         long newSnapshotTimestamp = Long.parseLong(metadataTable.get(txKey).getPayload().getVal());
         assertThat(lastAppliedSnapshot).isLessThan(newSnapshotTimestamp);
-        assertThat(newSnapshotTimestamp).isEqualTo(activeRuntime.getAddressSpaceView().getLogTail());
+//        assertThat(newSnapshotTimestamp).isEqualTo(activeRuntime.getAddressSpaceView().getLogTail());
```

Comment:
why is this no longer valid?

Reply:
It failed because the replication status is being updated a few more times.

Reply:
ok, I see. Then just delete this line!